presto-cassandra/src/test/resources/cu-cassandra.yaml

```diff
@@ -94,10 +94,10 @@ partitioner: org.apache.cassandra.dht.Murmur3Partitioner
 
 # directories where Cassandra should store data on disk.
 data_file_directories:
-    - target/embeddedCassandra/data
+    - ${data_directory}/data
```

Comment:
How is it related to `cassandra-unit` dependency (mentioned in the commit message)?